xml2kinto/__main__.py

```diff
@@ -137,6 +144,15 @@ def main(args=None):
     parser.add_argument('-x', '--xml-file', help='XML Source file',
                         type=str, default=XML_FILE)
 
+    parser.add_argument('-S', '--schema-file', help='JSON Schemas file',
+                        type=str, default=SCHEMA_FILE)
+
+    parser.add_argument('--no-schema', help='Should we handle schemas',
+                        action="store_true")
+
+    parser.add_argument('--with-scrapping', action="store_true",
+                        help='Activate blocklist scrapping on AMO')
```

Comment:
Actually it is relevant to the functional tests.
The scrapping can arm AMO and slow down functional tests by a lot. That's the reason why this is there.